### ISSUE TITLE
SYNCHDMLC gui not available in BEAMnrc gui

### ISSUE THREAD

**Issue by lwotzkeBCCA** [nrc-cnrc/EGSnrc#296]
It appears a gui has been written for the SYNCHDMLC component module, but it does not appear as an option when specifying a new accelerator in the BEAMnrc gui. This is despite being referred to in `beamnrc_gui.tcl` exactly the same way as other modules like as CONESTAK and FLATFILT. Is it possible this is because "synchdmlc" is 9 characters long? (CM Identifier is restricted to eight characters, maybe the same is true for CM Name.)

**Comment by rtownson** [comment]
In `HEN_HOUSE/omega/progs/gui/beamnrc/create_accel.tcl`, change line 191,
```
for {set i 1} {$i <= 24} {incr i} {
```
to:
```
for {set i 1} {$i <= [array size cm_names]} {incr i} {
```
Similarly for line 285. I'll submit this fix as a pull request.

**Comment by lwotzkeBCCA** [comment]
Thank you!

**Comment by rtownson** [comment]
This was fixed in PR #302.
